README.md

```diff
@@ -124,29 +140,34 @@ org.freedesktop.tuhi1.Device
       Otherwise, the device is paired and this function returns success (0)
       or a negative errno on failure.
 
-  Method: Listen() -> ()
-      Listen for data from this device. This method starts listening for
-      events on the device for an unspecified timeout. When the timeout
-      expires, a ListenComplete signal is sent indicating success or error.
+  Method: StartListening() -> ()
```

Comment:
Shouldn't this method return an error directly?

Reply:
true. this is a leftover thought from when listening meant "listen+sync" and had some long timeouts. Right now, the approach is safe for these timeouts, question is whether we want to leave this in. We can't really get rid of the `ListeningStopped` signal if we want to carry errors for unexpected shutdowns, so having the method return here doesn't really save on code.